app/Commands/GitHub/PrThreadsCommand.php

```
@@ -0,0 +1,282 @@
+            $this->line("<fg=yellow>Location:</> {$location}");
+        }
+        
+        $this->line("<fg=yellow>Participants:</> " . $thread->getParticipants()->implode(', '));
+        $this->line("<fg=yellow>Created:</> {$thread->createdAt->diffForHumans()}");
+        $this->line("<fg=yellow>Updated:</> {$thread->updatedAt->diffForHumans()}");
+
+        $this->newLine();
+        $this->line("<fg=cyan>Comments ({$thread->comments->count()}):</>");
+        $this->line(str_repeat('─', 50));
+
+        foreach ($thread->comments as $index => $comment) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$index'. (Unused Code Rules)

(UnusedLocalVariable)